src/crypto/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ set(crypto_plugin_dir ${CEPH_INSTALL_PKGLIBDIR}/crypto)
 
 add_subdirectory(openssl)
 
-if(HAVE_INTEL AND (HAVE_YASM_X64_AVX2 OR HAVE_NASM_X64_AVX2) AND (NOT APPLE))
+if(HAVE_INTEL AND HAVE_NASM_X64_AVX2 AND (NOT APPLE))
```

Comment:
might want to add the rationales in the commit message. i cannot find any context why we need this change by inspecting the comment or reading the commit message.

Reply:
@tchaikov 
I'll refine the commit message